src/examples/v2/mina-program.ts

```diff
@@ -0,0 +1,127 @@
+import { Bool, Experimental, Field, Int64, PrivateKey, PublicKey, UInt32, UInt64 } from 'o1js';
+
+const V2 = Experimental.V2;
+
+type TestState = {
+  x: typeof Field;
+  address: typeof PublicKey;
+};
+
+const TestState: Experimental.V2.StateDefinition<TestState> = V2.State({
+  x: Field,
+  address: PublicKey,
+});
+
+const TestProgram = V2.MinaProgram({
+  name: 'TestProgram',
+  State: TestState,
+  Event: V2.GenericData,
+  Action: V2.GenericData,
+
+  methods: {
+    init: {
+      privateInputs: [Field],
+
+      async method(
+        env: Experimental.V2.MinaProgramEnv<TestState>,
+        value: Field
+      ): Promise<Experimental.V2.MinaProgramMethodReturn<TestState>> {
```

Comment:
this is really awful but no other way if we want to keep it in the experimental namesapce for now